index.html

```diff
@@ -0,0 +1,215 @@
+    <div class="text-article">
+      <span><a href="">Text Article</a></span>
+    </div>
+    <section class="main-five">
+      <div>
+        <a href="">text</a>
+        <a href="">Button</a>
+        <a href="">Image</a>
+      </div>
+    </section>
+    <section class="main-six">
+      <div class="main-six-img"><a href="#">image</a></div>
+      <div class="main-six-text"><a href="#">text</a></div>
+    </section>
+    <section class="main-seven">
+      <div class="main-seven-grid1">
+        <div class="main-seven-img"><a href="#">image</a></div>
+        <div class="main-seven-text"><a href="#">text</a></div>
+      </div>
+      <div class="main-seven-grid2">
+        <div class="main-seven-img"><a href="#">image</a></div>
+        <div class="main-seven-text"><a href="#">text</a></div>
+      </div>
+      <div class="main-seven-grid3">
+        <div class="main-seven-img"><a href="#">image</a></div>
+        <div class="main-seven-text"><a href="#">text</a></div>
+      </div>
+      <div class="main-seven-grid4">
+        <div class="main-seven-img"><a href="#">image</a></div>
+        <div class="main-seven-text"><a href="#">text</a></div>
+      </div>
+      <div class="main-seven-grid5">
+        <div class="main-seven-img"><a href="#">image</a></div>
+        <div class="main-seven-text"><a href="#">text</a></div>
+      </div>
+      <div class="main-seven-grid6">
+        <div class="main-seven-img"><a href="#">image</a></div>
+        <div class="main-seven-text"><a href="#">text</a></div>
+      </div>
+    </section>
+    <div class="text-article2">
+      <span><a href="">Text Article</a></span>
+    </div>
+    <section class="main-eight">
+      <div>
+        <a href="" class="text">Text</a>
+        <a href="" class="img">Image</a>
+      </div>
+    </section>
+    <section class="main-nine">
+      <div class="main-nine-top">
+        <a href="#">Text</a>
+        <a href="#">Button</a>
+      </div>
+      <div class="main-nine-top">
+        <a href="#">Text</a>
+        <a href="#">Button</a>
+      </div>
+      <div class="main-nine-top">
+        <a href="#">Text</a>
+        <a href="#">Button</a>
+      </div>
+      <div class="main-nine-bottom">
+        <a href="#">Text</a>
+        <a href="#">Button</a>
+      </div>
+    </section>
+    <div class="text-article">
+      <span><a href="#">Text Article</a></span>
+    </div>
+  </main>
+  <footer>
+    <div class="footer-one">
+      <a href="#">Link</a>
+      <a href="#">Link</a>
+      <a href="#">Link</a>
+      <a href="#">Link</a>
+      <a href="#">Link</a>
+      <a href="#">Link</a>
+      <a href="#">Link</a>
+      <a href="#">Link</a>
+      <a href="#">Link</a>
+    </div>
+    <div class="footer-two">
+      <p>The last Text</p>
+    </div>
+    <div class="footer-three">
+      <a href="#">Image</a>
+      <div class="">
+        <a href="#">Link</a>
+        <a href="#">Link</a>
+        <a href="#">Link</a>
+      </div>
```

Comment:
- [ ] The links should be inside a `<ul>` tag and the links wrapped around `<li>` tags

Reply:
Thanks for the suggestions, made the changes accordingly.